samples/04_gis_analysts_data_scientists/export_video_using_arcgis_api_for_python.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>instead of manually  setting the index, we can load a map widget, zoom in and then run map.extent to let the user identify their area of interest. then the layer extent can be mapped with the help of map extent. for example look at https://github.com/Esri/arcgis-python-api/pull/636. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>fixed</p>